core/kazoo_apps/test/kapps_util_test.erl

```diff
@@ -0,0 +1,22 @@
+%%%-----------------------------------------------------------------------------
+%%% @copyright (C) 2013-2018, 2600Hz
+%%% @doc
+%%% @end
+%%%-----------------------------------------------------------------------------
+-module(kapps_util_test).
+
+-include_lib("eunit/include/eunit.hrl").
+-include_lib("kazoo_fixturedb/include/kz_fixturedb.hrl").
+
+kapps_util_with_fixtures_test_() ->
+    {'setup'
+    ,fun kzd_test_fixtures:setup/0
```

Comment:
you can use `kz_fixturedb_util:start_me/0,1` and `kz_fixutredb_util:stop_me/1` too.